neural_nets/activations/activations.py

```diff
@@ -22,6 +22,15 @@ def grad(self, x, **kwargs):
 
 
 class Sigmoid(ActivationBase):
+    """Softmax activation function.
```

Comment:
To maintain consistency with the rest of the repo, docstrings should be formatted according to the [NumPy styleguide](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)